types/web3/web3-tests.ts

```diff
@@ -22,3 +24,5 @@ const weiStr: string = web3.utils.toWei("100", "gwei");
 const weiBn: BigNumber = web3.utils.toWei(web3.utils.toBN("1"));
 const rndHex: string = Web3.utils.randomHex(10);
 const shhPromise: Promise<string> = web3.shh.generateSymKeyFromPassword("xyz");
+const storage: Promise<string> = web3.eth.getStorageAt("0xde0B295669a9FD93d5F28D9Ec85E40f4cb697BAe", 0);
```

Comment:
you can reuse `contractAddress` variable instead of a hardcoded address.

Reply:
Woops, that was the intention haha. Thanks for catching that

Reply:
I pushed the update, thanks again!